src/main/java/org/takes/http/BkBasic.java

```diff
@@ -122,17 +123,29 @@ public void accept(final Socket socket) throws IOException {
     private void print(final Request req, final OutputStream output)
         throws IOException {
         try {
-            new RsPrint(this.take.act(req)).print(output);
+            output.write(
```

Comment:
@baudoliver7 This could be done as `new BytesOf(new PsPrint(...)).bytes()`
Though it seems to be worth to make `RsPrint` implement `Bytes` itself (maybe as a separate todo)

Reply:
@andreoss Good ! I agree with you. Let's RsPrint implement Bytes itself.